src/main/java/wooteco/subway/domain/Fare.java

```diff
@@ -0,0 +1,35 @@
+package wooteco.subway.domain;
+
+public class Fare {
+
+    private static final int BASIC_FARE = 1250;
+    private static final int FIRST_RESTRICTION_DISTANCE = 10;
+    private static final int SECOND_RESTRICTION_DISTANCE = 50;
+    private static final double FIRST_RESTRICTION_DIVIDE = 5.0;
+    private static final double SECOND_RESTRICTION_DIVIDE = 8.0;
+    private static final int ADDITIONAL_FARE = 100;
+    private final int fare;
```

Comment:
상수와 필드는 개행으로 분리해두면 보기 더 편한거 같아요!
```suggestion
    private static final int ADDITIONAL_FARE = 100;
    
    private final int fare;
```